{
    "repository": "vslavik/bakefile",
    "clone_url": "https://github.com/vslavik/bakefile.git",
    "timestamp": "2023-05-29T14:01:08.655196Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[CI/build] \ud83d\ude80  Start image=crawlergpt:latest\n[CI/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[CI/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[CI/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[CI/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/vslavik-bakefile] user=0 workdir=\n[CI/build]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v2\n[CI/build] \u2b50 Run Main Check out\n[CI/build]   \u2705  Success - Main Check out\n[CI/build] \u2b50 Run Main Set up Python 2.7\n[CI/build]   \ud83d\udc33  docker cp src=/home/andre-silva/.cache/act/actions-setup-python@v2/ dst=/var/run/act/actions/actions-setup-python@v2/\n[CI/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /var/run/act/actions/actions-setup-python@v2/] user=0 workdir=\n[CI/build]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-python@v2/dist/setup/index.js] user= workdir=\n[CI/build]   \ud83d\udcac  ::debug::Semantic version spec of 2.7 is 2.7\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: \n[CI/build]   \ud83d\udcac  ::debug::explicit? false\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 2.7.18\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.5.10\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.6.14\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.7.11\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.8.11\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::isExplicit: 3.9.6\n[CI/build]   \ud83d\udcac  ::debug::explicit? true\n[CI/build]   \ud83d\udcac  ::debug::evaluating 6 versions\n[CI/build]   \ud83d\udcac  ::debug::matched: 2.7.18\n[CI/build]   \ud83d\udcac  ::debug::checking cache: /opt/hostedtoolcache/Python/2.7.18/x64\n[CI/build]   \ud83d\udcac  ::debug::Found tool in cache Python 2.7.18 x64\n[CI/build]   | Successfully setup CPython (2.7.18)\n[CI/build]   \u2753 add-matcher /run/act/actions/actions-setup-python@v2/.github/python.json\n[CI/build]   \u2705  Success - Main Set up Python 2.7\n[CI/build]   \u2699  ::set-env:: pythonLocation=/opt/hostedtoolcache/Python/2.7.18/x64\n[CI/build]   \u2699  ::set-env:: LD_LIBRARY_PATH=/opt/hostedtoolcache/Python/2.7.18/x64/lib\n[CI/build]   \u2699  ::set-output:: python-version=2.7.18\n[CI/build]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/2.7.18/x64\n[CI/build]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/2.7.18/x64/bin\n[CI/build] \u2b50 Run Main Install dependencies\n[CI/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/2] user= workdir=\n[CI/build]   | OCI runtime exec failed: exec failed: unable to start container process: error executing setns process: exit status 1: unknown\r\n[CI/build]   \u274c  Failure - Main Install dependencies\n[CI/build] Error response from daemon: Could not find the file /var/run/act/workflow/pathcmd.txt in container 6905538f60da2524ac7e7694b1eda46433217bac4e918e06dbb0bc267e2c21fd\n[CI/build] failed to remove container: Error response from daemon: No such container: 6905538f60da2524ac7e7694b1eda46433217bac4e918e06dbb0bc267e2c21fd\n[CI/build] \ud83c\udfc1  Job failed\n",
    "actions_stderr": "Error: Error occurred running finally: Error occurred running finally: Error response from daemon: remove act-CI-build-03727a8f260608c14dbfbecd23630d3cd3cc939c3150870ab3631218246ce38f-env: volume is in use - [c39322a782347b554506260cfc6b3d94b9b7b05815efcfb3b5d720307fecb6ef] (original error: <nil>) (original error: <nil>)\n"
}